config/configstore/jobs.go

```diff
@@ -0,0 +1,109 @@
+package configstore
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/centrifuge/go-centrifuge/ethereum"
+	"github.com/centrifuge/go-centrifuge/identity"
+	"github.com/centrifuge/go-centrifuge/jobs/jobsv2"
+	"github.com/centrifuge/gocelery/v2"
+	"github.com/ethereum/go-ethereum/common"
+)
+
+const (
+	generateIdentityRunnerName = "GenerateIdentity"
+	taskSendTxn                = "Create identity on Chain"
```

Comment:
I would add `Id` to the constant names `taskSendIdTxn` and `taskWaitForIdTxn` or alike.

Reply:
these are internal names, wont be used anywhere except this job. does Id has a significance ?